### DISCUSSION TITLE
CPU Usage

### DISCUSSION THREAD

**Discussion by DarinFlippo** [vincentneo/LosslessSwitcher#41] [Q&A]
The app seems to use some CPU (more than I'd expect...).  How often is it querying the logs?  Could this be configurable?  The code makes it appear to check every 2 seconds, but I may be wrong about that.  2 seconds seems pretty reasonable and shouldn't cause excessive CPU usage.

Great project - I've shared your repo on www.superbestaudiofriends.org where it is quite popular.

**Comment by vincentneo** [comment]
Thanks for sharing the repo!

I remember that for the beta (v1.1), I had indeed set it to 2s, after a track change has been detected, to run for 5 times, every 2s. As of now, it is not configurable.

For CPU usage, what's the percentage that you are observing? It should be the parsing of logs that used CPU, since in the betas it will check the logs for multiple keywords.
